whylog/config/parsers.py

```diff
@@ -10,8 +11,30 @@ def get_clue(self, line):
 
 
 class RegexParser(AbstractParser):
-    def __init__(self, name, regex, params):
-        pass
+    def __init__(self, name, regex, primary_key_groups, log_type, convertions):
+        self.name = name
+        self.regex_str = regex
+        self.primary_key_groups = primary_key_groups
+        self.log_type = log_type
+        self.convertions = convertions
 
     def get_clue(self, line):
         pass
+
+    def serialize_parser(self):
```

Comment:
If you use `__dict__`, then probably it will serialize just fine, without explicit calling of any method.